superset/advanced_data_type/plugins/eml_path.py

```diff
@@ -0,0 +1,81 @@
+import re
```

Comment:
I believe we want to store types in PB Gitlab now, since some types can't live in public Github.